src/targets/gpu/lowering.cpp

```diff
@@ -236,11 +240,37 @@ struct miopen_apply
             {
                 check_shape(s, insert_precompile_op(it));
             }
+            else if(attrs.contains("custom_op"))
+            {
+                check_shape(s, insert_custom_op(it, attrs));
+            }
         }
-
         copy_params();
     }
 
+    instruction_ref insert_custom_op(instruction_ref ins, value& attrs) const
```

Comment:
This should be `const value&`.

Reply:
yeah, but `operator[` doesn't work with `const` for now.

Reply:
made use of `.at()` to work with const.